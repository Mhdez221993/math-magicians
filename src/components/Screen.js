/* eslint-disable react/prop-types */
import React from 'react';

export default class Screen extends React.Component {
  constructor(props) {
    super(props);
    this.props = props;
  }

  render() {
    const {
      value: {
        next,
        total,
      },
    } = this.props;
    return (
      <div className="Display">
        <span className="Screen">{next || total || '0'}</span>
      </div>
    );
  }
}
