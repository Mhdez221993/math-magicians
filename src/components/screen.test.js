import React from 'react';
import renderer from 'react-test-renderer';
import Screen from './Screen';

it('Screen snapshot test', () => {
  const tree = renderer
    .create(<Screen value="0" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
