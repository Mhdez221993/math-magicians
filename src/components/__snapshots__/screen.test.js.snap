// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen snapshot test 1`] = `
<div
  className="Display"
>
  <span
    className="Screen"
  >
    0
  </span>
</div>
`;
